cypress/integration/authenticatedUserCanBecomeASubscriber.feature.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-undef */
+describe("User can see subscribe button", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/movies",
+      response: "fixture:movies_data.json",
+    });
+
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/subscriptions",
+      response: "fixture:stripe_response.json",
+    });
+
+    cy.visit("/");
+  });
+
+  describe("when not authenticated", () => {
+    it("is expected to be hidden", () => {
+      cy.get('[data-cy="index"]').within(() => {
+        cy.get('[data-cy="subscribe-button"]').should("not.be", "visible");
+      });
+    });
+  });
+
+  describe("when authenticated", () => {
+    it("is expected to be VISIBLE", () => {
```

Comment:
this message only indicates that the button should be visible, not that you can actually pay successfully.

Reply:
Yeah you are right! We missed that test. Thanks